app/cafe.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        check = visitor.get("vaccine", False)
```

Comment:
No need to set default value False because by default there is None that will not pass if statement below anyway